src/components/CodeView/index.spec.tsx

```diff
@@ -168,11 +168,9 @@ describe(__filename, () => {
   });
 
   it('handles empty content', () => {
-    const content = '';
-    const mimeType = 'text/css';
-    const root = renderWithLinterProvider({ mimeType, content });
+    const root = renderWithLinterProvider({ content: '' });
 
-    expect(root.find('.language-css')).toHaveProp('children', content);
+    expect(root.find('.tableBody')).toHaveProp('children', []);
```

Comment:
I guess this works because internally the lib adds this global class name. To make it more robust, this should use the `className` that we're explicitly setting:

```diff
diff --git a/src/components/CodeView/index.spec.tsx b/src/components/CodeView/index.spec.tsx
index 385b029..583de61 100644
--- a/src/components/CodeView/index.spec.tsx
+++ b/src/components/CodeView/index.spec.tsx
@@ -170,7 +170,7 @@ describe(__filename, () => {
   it('handles empty content', () => {
     const root = renderWithLinterProvider({ content: '' });
 
-    expect(root.find('.tableBody')).toHaveProp('children', []);
+    expect(root.find(`.${styles.tableBody}`)).toHaveProp('children', []);
   });
 
   it('renders multiple lines of code', () => {
```